src/pages/api/github-load-notes.ts

```diff
@@ -64,7 +64,9 @@ export async function loadNotesFromUrl(url: string) {
         }
       }, {})
       note.type = 'root'
-      const contents = toMarkdown(note)
+      const contents = note.children
+        .map((node: any) => toMarkdown(node))
+        .join(' ')
```

Comment:
Like Vojta said, only first node here was being recognised.

Reply:
@lsminter, I'd like to see a test that covers this circumstance. can you add that please.